src/services/importTracker.ts

```diff
@@ -582,6 +582,12 @@ namespace ts.FindAllReferences {
             return p.name !== node ? undefined :
                 p.parent.kind === SyntaxKind.CatchClause ? undefined : p.parent.parent.kind === SyntaxKind.VariableStatement ? p.parent.parent : undefined;
         }
+        else if (parent.parent && parent.parent.parent && parent.parent.parent.kind === SyntaxKind.VariableDeclaration) {

```

Comment:
does this work for binding elements of arbitrary nesting?

```ts
export const {foo: [bar, {baz: bas}]} = someObj;
```

You probably want to use `walkUpBindingElementsAndPatterns` to get the VariableDeclaration of the BindingElement.